docs/guides/setup.md

```diff
@@ -1,124 +1,87 @@
-Setup
-=====
+# Video.js Setup
 
-Video.js is pretty easy to set up. It can take a matter of seconds to get the player up and working on your web page.
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
 
-Step 1: Include the Video.js Javascript and CSS files in the head of your page.
-------------------------------------------------------------------------------
 
-You can download the Video.js source and host it on your own servers, or use the free CDN hosted version. As of Video.js 5.0, the source is [transpiled from ES2015](http://babeljs.io/) (formerly known as ES6) to [ES5](https://es5.github.io/), but IE8 only supports ES3. In order to continue to support IE8, we've bundled an [ES5 shim and sham](https://github.com/es-shims/es5-shim) together and hosted it on the CDN.
+- [Installing Video.js](#installing-videojs)
+- [Decorating a `<video>` Element](#decorating-a-video-element)
+  - [Automatic Setup](#automatic-setup)
+  - [Manual Setup](#manual-setup)
 
-```html
-<script src="//vjs.zencdn.net/ie8/1.1.1/videojs-ie8.min.js"></script>
-```
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
 
-### CDN Version ###
-```html
-<link href="//vjs.zencdn.net/5.4.6/video-js.min.css" rel="stylesheet">
-<script src="//vjs.zencdn.net/5.4.6/video.min.js"></script>
-```
+## Installing Video.js
 
-Alternatively you can always [go here](http://videojs.com/getting-started/) to get the latest URL for videojs CDN.
+Video.js is officially available via CDN, npm, and Bower.
 
-We include a stripped down Google Analytics pixel that tracks a random percentage (currently 1%) of players loaded from the CDN. This allows us to see (roughly) what browsers are in use in the wild, along with other useful metrics such as OS and device. If you'd like to disable analytics, you can simply include the following global **before** including Video.js:
```

Comment:
Actually, I think we should move this to the "Getting Started" guide. It's silly to have two documents covering the same thing.